src/components/picker/index.tsx

```diff
@@ -93,6 +96,19 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
   } = themeProps;
   const {preset} = others;
 
+  const {
+    type,
+    headerProps,
+    renderCustomModal: typeRenderCustomModal,
+    dialogProps,
+    pickerModalProps
+    //@ts-expect-error
+  } = usePickerMode({
+    pickerType: 'modal',
+    ...themeProps
+  });
+  const isDialog = useDialog || type.dialog;
```

Comment:
Type wheelPicker is also a Dialog, no?

Reply:
`WheelPicker` does use Dialog eventually (from `ExpandableOverlay`), but the difference is that `WheelPicker` renders inside `PickerItemList` as `<WheelPicker/>` component and handeling the items by itself so it should be separated.